### PR TITLE
OCI Helm Registries - Sync Supported

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
## Describe the Change

This PR updates the documentation to reflect synchronization support for OCI Helm registries. 

## Changed Pages

💻 [OCI Index Page](https://deploy-preview-4181--docs-spectrocloud.netlify.app/registries-and-packs/registries/oci-registry/)

💻 [Add OCI Helm](https://deploy-preview-4181--docs-spectrocloud.netlify.app/registries-and-packs/registries/oci-registry/add-oci-helm/)

## Jira Tickets

🎫 [PEM-5472](https://spectrocloud.atlassian.net/browse/PEM-5472)

## Backports

Can this PR be backported?

- [ ] Yes. _Remember to add the relevant backport labels to your PR._
- [x] No. _Please leave a short comment below about why this PR cannot be backported._
Release PR

[PEM-5472]: https://spectrocloud.atlassian.net/browse/PEM-5472?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/docs-content/registries-and-packs/registries/oci-registry/oci-registry.md [hunk 1]
```diff
@@ -27,10 +27,10 @@ The following type of OCI registries are supported:
 
 ## Limitations
 
-- Public OCI registries are not supported. You must use a private OCI registry that supports basic authentication.
+- Synchronization is not supported for Zarf registries.
```
Comment (caroldelwing) [inline docs/docs-content/registries-and-packs/registries/oci-registry/oci-registry.md:30]: Just to confirm: Synchronization or automatic synchronization? :)

**Review by caroldelwing — APPROVED** [review]
:rocket: Just one tiny question.

File: docs/docs-content/registries-and-packs/registries/oci-registry/oci-registry.md (hunk 1 above)
Comment (karl-cardenas-coding) [reply docs/docs-content/registries-and-packs/registries/oci-registry/oci-registry.md:30]: Good catch @caroldelwing

Comment (karl-cardenas-coding) [inline docs/docs-content/registries-and-packs/registries/oci-registry/oci-registry.md:30]:
```suggestion
- Automatic Synchronization is not supported for Zarf registries.
```

**Pull request merged**